packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -186,6 +190,21 @@ class Transforms {
           activity.participantDisplayNames = space.participantDisplayNames.filter((value) => value !== activity.actorDisplayName).toString();
         }
 
+        // only post and share activities have content which needs to be decrypted
+        if (activity.verb !== 'post' && activity.verb !== 'share') {
+          return Promise.resolve(object);
+        }
+
+        if (!activity.encryptionKeyUrl) {
+          // The encryptionKeyUrl may not be necessary if the activity content is unencrypted, so we treat it as a warning, rather than an error
+          const reason = `Activity ${activity.activityId} cannot be decrypted due to a missing encryption key url`;
+
+          ctx.webex.logger.warn(reason);
+          activity.warn = reason;
+
+          return Promise.resolve(object);
```

Comment:
This could be a warning or an error but you always treat it as a warning. i.e. As it may be plaintext or encrypted.

I think anything we can't decrypt is an error as the cloud is meant to be using encryption for everything.

Reply:
👍